contrib/masterfiles/git-failsafe.cf

```diff
@@ -198,7 +198,7 @@ bundle agent git_update
    "$(inputs_dir)/cf_promises_validated"
         comment => "Check whether a validation stamp is available for a new policy update to reduce the distributed load",
          handle => "check_valid_update",
-      copy_from => u_rcp("$(master_location)/cf_promises_validated","$(sys.policy_hub)"),
+      copy_from => u_rcp("$(master_location)/cf_promises_validated","$(sys.policy_hub_ip)"),
```

Comment:
Sounds good! Then just leave the entries that are relevant to ACLs, and change back the `copy_from` ones to `policy_hub`.